app/src/main/java/org/mozilla/focus/fragment/onboarding/OnboardingSecondFragment.kt

```diff
@@ -71,4 +79,9 @@ class OnboardingSecondFragment : Fragment() {
             }
         }
     }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        activity?.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED
```

Comment:
If this already happens in `handleFinishOnBoarding` do we still need it here?

Reply:
Done